### ISSUE TITLE
Tables in Go documentation are awkwardly formatted

### ISSUE THREAD

**Issue by jjlin** [golang/go#67368]
### What is the URL of the page with the issue?

https://go.dev/ref/mod#environment-variables

### What is your user agent?

Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/124.0.0.0 Safari/537.36

### Screenshot

![image](https://github.com/golang/go/assets/203380/358aae01-98cb-4338-9299-6711f4e87011)

### What did you do?

Visit https://go.dev/ref/mod#environment-variables

### What did you see happen?

The table of environment variables has no cell borders and the content is vertically centered. This makes it needlessly difficult to understand which variable corresponds to which description, and where one description ends and the next begins.

### What did you expect to see?

If the table is going to have no borders, then I think the content should be top-aligned.

I'd expect that other tables in the Go documentation probably have this issue as well.

**Comment by hyangah** [comment]
I agree this table will look better and more readable if it's top-aligned.
CSS adjustment for this table is welcome.

**Comment by heysujal** [comment]
Hi, I would like to work on this issue and want to know if you assign issues here.
I have found the file `ref/mod.md` in which changes need to be made.
I wanted to know how you are applying css to the table ?
The table is having a class `ModTable`

**Comment by kevinpeno** [comment]
Not sure where to throw it, but the following worked well for that page, imo. I didn't see `.ModTable`   defined anywhere in styles.css, so there shouldn't be any conflicts.

```css
.ModTable th, .ModTable td {
    vertical-align: baseline;
    text-align: left;
    padding: 1rem;
}
```

**Comment by deitrix** [comment]
Bringing over my two cents from the duplicate issue. Alternating the row's background colour is another way to go.

![Image](https://github.com/user-attachments/assets/509a145f-8bb6-45e8-9ecf-e3694275f3d9)

**Comment by adonovan** [comment]
Now:

<img width="713" height="1365" alt="Image" src="https://github.com/user-attachments/assets/02bd2892-2170-4fa4-8ff7-af2050d1e98f" />
<img width="1070" height="864" alt="Image" src="https://github.com/user-attachments/assets/c3636be3-c134-4db2-8e21-353c8bc3ec2c" />
